Description: Disables the Maven publish plugin
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -1,5 +1,4 @@
 apply plugin: 'java-library'
-apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
 group = 'org.dom4j'
@@ -56,6 +55,7 @@
     from javadoc.destinationDir
 }
 
+/*
 publishing {
     publications {
         mavenJava(MavenPublication) {
@@ -101,6 +101,7 @@
         }
     }
 }
+*/
 
 jar {
     manifest {
@@ -112,6 +113,7 @@
     useTestNG()
 }
 
+/*
 if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword')) {
     publishing {
         repositories {
@@ -133,6 +135,7 @@
         }
     }
 }
+*/
 
 if (project.hasProperty('signing.keyId')) {
     signing {
